src/sync/list.rs

```diff
@@ -46,15 +48,17 @@ impl<T> Node<T> {
     /// Returns the data in this entry.
     fn new(data: T) -> Self {
         Node(CachePadded::new(NodeInner {
-            data: data,
+            data,
```

Comment:
Is this intentional? Why do we ignore `data`?

Reply:
It's not ignored -- this is just the *field init shorthand* feature:
https://www.reddit.com/r/rust/comments/5udcju/field_init_shorthand_stabilization_just_landed/